utils/db.js

```diff
@@ -14,3 +15,17 @@ WHERE article_Id =$1`,
       }
     });
 };
+
+exports.checkUserExist = (table, column, value) => {
```

Comment:
I like this but seems to be kept generic but has name and promise reject specific to users, can this either be generalised or be amended so it is specific to users?